### DISCUSSION TITLE
Readme instructions

### DISCUSSION THREAD

**Discussion by ismaail** [gwleuverink/relay#1] [Ideas]
To contribute with PR or report an Issue, Please provide (in Readme file) instructions to clone and setup the project in order to try it.

**Comment by gwleuverink** [comment]
Hey 👋 This was a silent release. Was still working out some minor build issues. 

You can download a working build here: https://github.com/gwleuverink/relay/releases

> Please provide (in Readme file) instructions to clone and setup the project

I will update the readme (and maybe add a makefile) before I communicate about the app publicly. In the mean time, if you want to poke around:

``` bash
// clone the project

cp .env.example .env

composer install
npm i

php artisan key:generate
php artisan native:install # install dependencies - don't start dev server

composer native:build
```

**Comment by ismaail** [comment]
Thanks for this instructions.

I use Linux and there's no AppImage in the release to try. 

But mainly since I'm new to NativePHP, I want to learn it more by viewing and contributing to Open Source code like this one.

**Comment by ismaail** [comment]
I manager to run it locally, I figured I need to create a **Github App** to get **GITHUB_CLIENT_ID**.

I sent a PR to fix file path.
